engine/hatchery/swarm/swarm.go

```diff
@@ -73,6 +74,23 @@ func (h *HatcherySwarm) SpawnWorker(spawnArgs hatchery.SpawnArguments) (string,
 
 	var network, networkAlias string
 	services := []string{}
+	if network == "" {
+		hn, err := os.Hostname()
+		if err != nil {
+			return "", err
+		}
+		if nl, err := getContainerNetworks(h.dockerClient, hn); err != nil {
+			return "", err
+		} else {
```

Comment:
if block ends with a return statement, so drop this else and outdent its block (move short variable declaration to its own line if necessary)